cmd/ttn-lw-cli/commands/simulate.go

```diff
@@ -242,160 +248,163 @@ func simulate(cmd *cobra.Command, forUp func(*ttnpb.UplinkMessage) error, forDow
 	return ctx.Err()
 }
 
-func processDownlink(dev *ttnpb.EndDevice) func(lastUpMsg *ttnpb.Message, downMsg *ttnpb.DownlinkMessage) error {
-	return func(lastUpMsg *ttnpb.Message, downMsg *ttnpb.DownlinkMessage) (err error) {
-		phy, err := band.GetByID(dev.FrequencyPlanID)
-		if err != nil {
-			return err
+func processDownlink(dev *ttnpb.EndDevice, lastUpMsg *ttnpb.Message, downMsg *ttnpb.DownlinkMessage) error {
```

Comment:
Very unfortunate diff here, but had to do this to use the `n_f_cnt_down` here correctly. This function returning a closure adds nothing, but unnecessary complexity anyway.